.openshift-ci/ci_tests.py

```diff
@@ -86,75 +85,14 @@ def set_dirs_after_start():
 
 
 class OperatorE2eTest(BaseTest):
-    # TODO(ROX-12348): adjust these timeouts once we know average run times
-    FETCH_KUTTL_TIMEOUT_SEC = 5 * 60
-    DEPLOY_TIMEOUT_SEC = 40 * 60
-    UPGRADE_TEST_TIMEOUT_SEC = 50 * 60
-    E2E_TEST_TIMEOUT_SEC = 50 * 60
-    SCORECARD_TEST_TIMEOUT_SEC = 20 * 60
-
-    def __init__(self):
-        self.test_outputs = [
-            "operator/build/kuttl-test-artifacts",
-            "operator/build/kuttl-test-artifacts-upgrade",
```

Comment:
I assume `self.test_results` are now taken care of by the `store_test_results` calls in the script, but why get rid of these?

Reply:
> I assume `self.test_results` are now taken care of by the `store_test_results` calls in the script, but why get rid of these?

Because they are now taken care of by store_test_results.